protected/views/adminSample/_form.php

```diff
@@ -65,10 +67,82 @@
 
         <div class="pull-right btns-row">
             <a href="/adminSample/admin" class="btn background-btn-o">Cancel</a>
-            <?php echo CHtml::submitButton($model->isNewRecord ? 'Create' : 'Save', array('class' => 'btn background-btn')); ?>
+            <button id='checkAttribute' type='button' class="btn btn-primary"><?php echo $model->isNewRecord ? 'Create' : 'Save'?></button>
+
         </div>
 
+        <div class='modal fade' id='confirmation_sample_modal' role='dialog'>
+            <div class='modal-dialog modal-lg'>
+                <div class='modal-content'>
+                    <div class='modal-header'>
+                        <button type='button' class='close' data-dismiss='modal'>&times;</button>
+                        <h4 class='modal-title'>Important</h4>
+                    </div>
+                    <div class='modal-body'>
+                        <div id='check-attribute-warning' class='alert alert-warning' style='display: none;'>
+                        </div>
+                        <div id="check-attribute-confirmation" class="mt-4">
+                        </div>
+                    </div>
+                    <div class='modal-footer'>
+                        <a id="hideModal" class='btn background-btn-o'>Cancel</a>
+                        <?php echo CHtml::submitButton('Confirm', array('class' => 'btn background-btn')); ?>
```

Comment:
**question:** If the invalid geolocation was allowed to be overrided and saved, what is the point of having the validation rule?
**suggestion:** My suggestion is not allow user to save any invalid value.

Reply:
I agree with you but it has been requested to make it just a warning message that can be overriden

Reply:
As a user (and product owner) I think we should set it as a hard rule with no over-ride. It wont stop errors in format coming in via the excel spreadsheet upload but it will make sure curators fix them if they try to edit the sample metadata.

Reply:
I have conflicting information regarding whether to block the action or just display a warning message